core/src/main/scala/flow/FlowBuilder.scala

```diff
@@ -26,7 +26,7 @@ import collection.immutable.{Iterable, Seq}
   *   .flatMap {
   *     case i if i < 0 => Failure(new Exception("number is negative!"))
   *     case i => Success(i)
-  *   }.mapWithContext((i, context, metadata) => s"Value: $i, Context: $context")
+  *   }.mapWithContext((i, context, metadata) => s"Value: $$i, Context: $$context")
```

Comment:
Why did this need to change?

Reply:
the local compiler was trying to do string interpolation inside the comment and failed. Not sure if that's the case with our pipeline compiler, but it was failing locally.

Reply:
Ohhhh I didn't notice this was in a comment. Lol yeah this is fine then 👍